packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -46,28 +46,28 @@ export const CustomScrollbar: FC<Props> = ({
   children,
 }) => {
   const ref = useRef<Scrollbars & { view: HTMLDivElement }>(null);
+  const styles = useStyles2(getStyles);
+
   useEffect(() => {
-    if (ref.current) {
-      scrollRefCallback?.(ref.current.view);
+    if (ref.current && scrollRefCallback) {
+      scrollRefCallback(ref.current.view);
     }
   }, [ref, scrollRefCallback]);
-  const styles = useStyles2(getStyles);
 
-  const updateScroll = () => {
-    if (ref.current && !isNil(scrollTop)) {
+  useEffect(() => {
+    if (ref.current && scrollTop != null) {
+      if (ref.current.view.scrollTop === scrollTop) {
+        console.log('CustomScrollbar: update but  the same');
+      }
+
       ref.current.scrollTop(scrollTop);
     }
-  };
-
-  useEffect(() => {
-    updateScroll();
-  });
+  }, [scrollTop]);
```

Comment:
The change makes sure we only update scroll position when scrollTop changes (from outside), before the PR there was not real check that that scrollTop has actually changed (since it was not in the useEffect dependency list)